test/native/callback-api-tests.js

```diff
@@ -8,6 +8,7 @@ test('fires callback with results', function() {
   client.query('SELECT 1 as num', assert.calls(function(err, result) {
     assert.isNull(err);
     assert.equal(result.rows[0].num, 1);
+    assert.equal(result.rowCount, 1);
```

Comment:
I think you _might_ want to use `assert.strictEquals` here - doing an `assert.equals` basically does ` "1" == 1` which is true. What do you think?